morebits.css

```diff
@@ -8,22 +8,60 @@
  * The latest development source is available at [https://github.com/wikimedia-gadgets/twinkle/blob/master/morebits.css].
  */
 
+/* Define CSS variables */
+html {
+	--morebits-color-status-status: #4682B4;
+	--morebits-color-legend: #31628F;
+	--morebits-color-tooltip: #0000cd;
+	--morebits-color-info: #228B22;
+	--morebits-color-warning: #FF4500;
+	--morebits-color-titlebar-links: #3062AD;
+	--morebits-bgcolor-dialog: #F0F8FF;
+	--morebits-bgcolor-titlebar: #BCCADF;
+}
+
+@media screen {
+	html.skin-theme-clientpref-night {
+		--morebits-color-status-status: #A6C4DE;
+		--morebits-color-legend: #6B9DCC;
+		--morebits-color-tooltip: #7A7AFF;
+		--morebits-color-info: #6BDB6B;
+		--morebits-color-warning: #FFC7B3;
+		--morebits-color-titlebar-links: #8BADDF;
+		--morebits-bgcolor-dialog: #17244A;
```

Comment:
This is subjective, but I think the blue on black looks terrible. Might want to use a shade of grey instead.

Reply:
Sorry, which color are you talking about?

Reply:
morebits-bgcolor-dialog

Reply:
I experimented with some other colours, and I think this might be better (screenshot attached):

- bgcolor-dialog: #1b1b30
- bgcolor-titlebar: #1c2a52

![image](https://github.com/user-attachments/assets/ce72b340-5560-42cf-9eb6-45995a059685)

Reply:
I think it's better to reuse `--background-color-neutral` or `--background-color-neutral-subtle` for the dialog background. Titlebar colour seems okay.

Reply:
I disagree: background-color-neutral-subtle (and the other one) don't fit the blue aesthetic of twinkle. There needs to be a touch of blue, otherwise it doesn't feel like Twinkle and creates a weird contrast between the titlebar and dialog.
![image](https://github.com/user-attachments/assets/0f40c85b-b871-4ce7-89d4-6c5abba3c0e7)

Reply:
Choice of colors is dependent on the theme. Pale blue dialog against a white background looks nothing like this gawdy dark blue dialog on a black background. If the contrast with the titlebar is a problem, that can be changed as well.

Reply:
We still need some kind of accent colour to differentiate Twinkle from other content. Maybe a more subtle dark blue is better?

- bgcolor-dialog: #141c26
- bgcolor-titlebar: #1c2a52

![image](https://github.com/user-attachments/assets/a8dbb503-4f96-44a5-9299-9bfadf80aabe)

Reply:
Yes, this looks okay to me.